cmd/runtimetest/main.go

```diff
@@ -204,6 +204,39 @@ func validateSysctls(spec *specs.LinuxSpec, rspec *specs.LinuxRuntimeSpec) error
 	return nil
 }
 
+func validateMount(spec *specs.LinuxSpec, rspec *specs.LinuxRuntimeSpec) error {
+	fmt.Println("validating mount")
+	//read the /proc/mount file and covert to map[path]spec.mount
```

Comment:
On Mon, Jan 25, 2016 at 03:18:48AM -0800, Wang Qilin wrote:
> 1. about required mounts check, I think it's more appropriate to
> finish it in bunlde validator , not in runtime test. That means if
> you do not load these required mounts, your bundle is considered
> `illegal`

As I read opencontainers/specs#164, the *runtime* must supply those
mounts regardless of the user config (unless we specify an opt-out
mechanism, which we don't have yet).  So it's a runtime-author problem
(not a bundle-author problem) if they're missing.

> 2. I agree with your idea about not validating any bundle's mounts,
> but there is a problem that in current testing framework , we use a
> single bundle to test all `specs` items , so we need to cover the
> restrictions defined in `specs`, like `type` in mounts [1]

It doesn't seem possible to test different config permutations with a
single bundle.  You could probably do it with a single rootfs and a
bunch of per-test configs (which was how I understood this repository
to work).  If setting up a bundle to bind-mount a file from the host
into the container is too difficult to do, we should probably adjust
ocitools to make it easier ;).